### PR TITLE
enable metrics expire for replica selection

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
 issue: #25341

### CODE REVIEW

File: internal/proxy/look_aside_balancer.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ import (
 
 var (
 	checkQueryNodeHealthInterval = 500 * time.Millisecond
+	CostMetricsExpireTime        = 1000 * time.Millisecond
```
Comment (jiaoew1991) [inline internal/proxy/look_aside_balancer.go:40]: Do we need it to be configurable?

Comment (jiaoew1991) [reply internal/proxy/look_aside_balancer.go:40]: the same as checkQueryNodeHealthInterval

Comment (weiliu1031) [reply internal/proxy/look_aside_balancer.go:40]: the internal policy, config it's value is no meaning for users.

Comment (jiaoew1991) [reply internal/proxy/look_aside_balancer.go:40]: It is not really useful for end users, mainly during the operation and maintenance phase. Will adjusting parameters be helpful when problems occur? 🤔

Comment (weiliu1031) [reply internal/proxy/look_aside_balancer.go:40]: `CostMetricsExpireTime` doesn't take effect during problems occur. it just worked in multi-replica cases with very low search/query workload.

`checkQueryNodeHealthInterval` will decide the latency that proxy knows query node is unavailable. so to reduce the latency, we set a nearly smallest and reasonable value here. give it a larger value doesn't make sense

**Pull request merged**